core/shared/src/main/scala/org/threeten/bp/Duration.scala

```diff
@@ -89,7 +89,7 @@ object Duration {
    * @throws ArithmeticException
    *   if the input days exceeds the capacity of { @code Duration}
    */
-  def ofDays(days: Long): Duration = create(Math.multiplyExact(days, 86400), 0)
+  def ofDays(days: Long): Duration = create(Math.multiplyExact(days, 86400L), 0)
```

Comment:
This is a bit odd, is the auto conversion not workin?

Reply:
Yep, it doesn't seem to work in Scala Native :shrug: